### ISSUE TITLE
XIVALEXANDER EVEN WORKING ATM?

### ISSUE THREAD

**Issue by madara4567** [Soreepeong/XivAlexander#456]
I'm on console, and the thing just ain't working anymore. Keep getting errors while loading. Like "ffxiv has to be in same directory". "incorrect header".

ANYONE can confirm the app is having issues since this patch?

**Comment by Mayushii-chan** [comment]
Wrong repository, but yes, it is working.
https://github.com/Soreepeong/XivMitmLatencyMitigator

Follow the new instructions to get it working again and if you run into any problems you can probably look in "Issues" for a solution.

**Comment by madara4567** [comment]
> Wrong repository, but yes, it is working. https://github.com/Soreepeong/XivMitmLatencyMitigator
> 
> Follow the new instructions to get it working again and if you run into any problems you can probably look in "Issues" for a solution.

"Copy ffxiv.exe from your local Windows/Mac game installation into the VM"

Wait what? So I need to download and install FFXIV entirely on PC now to get it working on console? Do I have that right?

**Comment by Mayushii-chan** [comment]
Yes, you need the "ffxiv.exe" to get it working again because of the changes SE did in 6.15.

You can download the free trial to get it, or maybe ask a friend for it.

**Comment by madara4567** [comment]
"Copy ffxiv.exe from your local Windows/Mac game installation into the VM."
I  have no idea how to even do this. Does it mean copy it and put it inside the VM folder? Very confused about that.

**Comment by madara4567** [comment]
Yep, getting a file copied into there seems to be extremely troublesome..I wonder how many people are just not able to figure it out.

**Comment by madara4567** [comment]
The download is failing in arch linux for some inexplicable reason because i can get the file via opening browser on HFS. Gotta love this shit..

**Comment by madara4567** [comment]
download fails on Arch linux. Got it done on Ubuntu, but then curl/Soreepeong,etc is still giving no permission lol
